quarkus/service/src/test/java/org/apache/polaris/service/quarkus/admin/ManagementServiceTest.java

```diff
@@ -132,6 +134,7 @@ public void testUpdateCatalogWithDisallowedStorageConfig() {
             .catalogsApi()
             .createCatalog(
                 new CreateCatalogRequest(catalog),
+                Mockito.mock(HttpHeaders.class),
```

Comment:
would be really nice if we can add some e2e test, for this ?

Reply:
Yeah I would love to, but there's nothing actually consuming the headers right now... in a sense, the template change is the actual change, and the test is that we needed to update all the other files in order to make things pass.